connector/connect/client/jvm/src/main/scala/org/apache/spark/sql/connect/client/GrpcStub.scala

```diff
@@ -0,0 +1,51 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.apache.spark.sql.connect.client
+
+import io.grpc.ManagedChannel
+
+import org.apache.spark.connect.proto.{AnalyzePlanRequest, AnalyzePlanResponse, ConfigRequest, ConfigResponse, ExecutePlanRequest, ExecutePlanResponse, InterruptRequest, InterruptResponse}
+import org.apache.spark.connect.proto
+
+private[client] class GrpcStub(channel: ManagedChannel) {
```

Comment:
My last nit maybe better name is CustomSparkConnectBlockingStub?

Reply:
or better SparkConnectCustomBlockingStub? :stuck_out_tongue_winking_eye:

Reply:
🤷

Reply:
Thanks for the suggestions. Both are better names. The naming convention for the generated SparkConnectService grpc stub seems to be the concatenantion of s"SparkConnectService(Blocking|Future|...)Stub". So SparkConnectCustomBlockingGrpcStub seems to be more grpc idiomatic